styles.css

```diff
@@ -1,3 +1,335 @@
-* {
-    font-size: 1.5rem;
+li a {
+    font-family: "Bakbak One", serif;
```

Comment:
Provide fallback fonts and family types at the end of the font list. 

For example: `font-family: "Roboto", Arial, sans-serif;`